.env.example

```
@@ -15,9 +15,7 @@ NEXT_PUBLIC_API_URL=https://nodes-api-dev.desci.com
 # Allows testing local env with a prod API, but carries some security risk due to loosened cookie security settings, should only be enabled in dev mode.
 NEXT_PUBLIC_LOCAL_AUTH_COOKIE=true
 
-DESCI_RESEARCH_CONNECTION_DATABASE_URL=""
-
 NEXT_PUBLIC_BASE_URL=http://localhost:3000
 
 SCIWEAVE_SUPABASE_URL=
-SCIWEAVE_SUPABASE_SERVICE_KEY=
\ No newline at end of file
+SCIWEAVE_SUPABASE_SERVICE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SCIWEAVE_SUPABASE_SERVICE_KEY key should go before the SCIWEAVE_SUPABASE_URL key

(UnorderedKey)